pitest-maven/src/main/java/org/pitest/maven/MojoToReportOptionsConverter.java

```diff
@@ -154,14 +155,23 @@ private ReportOptions parseReportOptions(final List<String> classPath) {
     return data;
   }
 
+  private void configureVerbosity(ReportOptions data) {
+    if (this.mojo.isVerbose()) {
+      data.setVerbosity(Verbosity.VERBOSE);
+    } else {
+      Verbosity v = Verbosity.fromString(mojo.getVerbosity());
```

Comment:
Am I thinking right that if the configuration explicitly says "verbose = false" but "verbosity=verbose", it will be verbose, and if it says "verbose = true" but "verbosity=QUIET" it will be also be verbose? Should it be considered an error to explicitly set both?

Reply:
Will probably remove the verbose option in a couple of releases time. In the meantime this approach lets people upgrade without changing their build scripts. 

Don't think the argument that this is better than erroring is particularly strong, but I don't think the counter argument is any stronger. Ultimately both behaviours are sensible, but arbtrary so makes sense to stick with the one that's already coded.